Kickstarter-iOS/Views/Controllers/ToggleViewControllerTests.swift

```diff
@@ -0,0 +1,58 @@
+@testable import Kickstarter_Framework
+@testable import Library
+import UIKit
+
+final class ToggleViewControllerTests: TestCase {
+  override func setUp() {
+    super.setUp()
+
+    AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    UIView.setAnimationsEnabled(false)
+  }
+
+  override func tearDown() {
+    AppEnvironment.popEnvironment()
+    UIView.setAnimationsEnabled(true)
+
+    super.tearDown()
+  }
+
+  func testView() {
+    let devices = [Device.phone4_7inch, Device.pad]
+    combos([Language.en], devices).forEach { language, device in
+      withEnvironment(language: language) {
+        let controller = ToggleViewController.instantiate()
+        controller.titleLabel.text = "Title for testing purposes only"
```

Comment:
Since the base class doesn't provide default title or ON/OFF state we're setting it here simply for the purposes of our tests.